CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## [0.0.1-alpha.25](https://github.com/stackblitz/tutorialkit/compare/0.0.1-alpha.23...0.0.1-alpha.25) (2024-07-09)
```

Comment:
@Nemikolh any ideas why this comparison (and changelogs itself) start from `23` and not `24`?

Reply:
This is because the tag was never pushed. If we push the `0.0.1-alpha.24` tag and re-run the changelog it should work

Reply:
Oh I just noticed that there's a bug in our workflow. It push a tag with `v` in front causing this to fail.

Reply:
The `v` is intended 😄 
When you run `npm version patch`, it adds `v` prefix. 

```bash
ari ~/Git/aria-live-capture (master) $ npm version patch
v2.0.1
ari ~/Git/aria-live-capture (master) $ git tag
v2.0.0
v2.0.1
```

Looking at docs of `conventional-commits` I think it uses that as default, right? https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-core#tagprefix